content/guides/erp-connection/step1-who-is-your-connector-for.md

```diff
@@ -0,0 +1,99 @@
+# Who is your connector for?
+
+## Personas
+
+Below, you will find the user archetypes we use to help you guide your decisions about product features, interactions, and even the visual design of your connector. Designing for the archetype — whose goals and behavior patterns are well understood — allows you to satisfy the broader group of people represented by that archetype.
+
+![Personas](../../img/guides/personas.png)
+
+## Product workflow overview
+
+The organization of Julia's work on products can be done in 2 different ways when using an ERP and Akeneo PIM.
+
+### The product is initially created in the ERP
+
+With this organization, the product is initially created by ERP teams, so it contains mainly technical information.
+
+As an image is worth a thousand words, here's an example of a product workflow with the ERP, Akeneo PIM and an eCommerce solution:
+![Products initially created in the ERP](../../img/guides/erp-pim-ecommerce-flow.svg)
+
+Julia's needs will be to:
+1. Retrieve product information from the ERP
+2. Organize, enrich, translate product information in her PIM
+3. Export these PIM data to different channels such as an e-commerce website, a print solution, a mobile application...
+
+### The product is initially created in the PIM
+
+In this organization variation, the product workflow is slightly different: this time, the product is initially created by Julia (and no longer by the ERP teams):
+![Products initially in the PIM](../../img/guides/pim-erp-ecommerce-flow.svg)
+
+Julia's needs will be to:
+1. Create the product in her PIM with a SKU and some basic marketing information (such as the product name, short description)
+2. Export this product information from her PIM to the ERP solution to allow the ERP teams to add technical product information (such as price, stock information, automatic categorization...)
+3. Import again part of the ERP product information (ERP data that need to be enriched or used by Julia)
+4. Finalize the product marketing information in her PIM
+5. Export these PIM data to different channels such as an e-commerce website, a print solution, a mobile application...
+
+:::warning
+The initial PIM action to create new products in the PIM requires Julia to have a process to **automatically** generate products SKU directly in the PIM.
```

Comment:
The phrase "to have a process to **automatically** generate products SKU directly in the PIM" , could be taken as it is actually doable on the PIM, and by now there is not sku generator feature on the pim. 

Something more ambiguous like :  
to have a process to **automatically** generate products SKU and then assign this into the PIM